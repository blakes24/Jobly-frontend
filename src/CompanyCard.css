.CompanyCard {
  -webkit-box-shadow: 0px 1px 2px 0px lightgrey;
  box-shadow: 0px 1px 2px 0px lightgrey;
  margin: 10px;
  border-radius: 0;
}

.CompanyCard-title {
  display: flex;
  justify-content: stretch;
  position: relative;
}

.CompanyCard-title .card-title {
  padding-right: 30px;
}

.CompanyCard-title img {
  position: absolute;
  right: 0;
  top: 0;
}

.CompanyCard a {
  color: black;
}

.CompanyCard a:hover {
  color: #0069d9;
  text-decoration: none;
  -webkit-box-shadow: 0px 3px 10px 2px lightgrey;
  box-shadow: 0px 3px 10px 2px lightgrey;
}

.CompanyCard .card-title {
  position: relative;
}

.CompanyCard img {
  position: absolute;
  right: 0;
}

@media screen and (max-width: 600px) {
  .CompanyCard .card {
    margin-top: 1rem;
    box-shadow: none;
    border: none;
  }
}
