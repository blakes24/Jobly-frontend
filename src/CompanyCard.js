import { Card } from 'react-bootstrap';

const CompanyCard = ({ item }) => {
  return (
    <Card>
      <Card.Body>
        <Card.Title>
          {item.name} <img src={item.logoUrl} alt="logo" />
        </Card.Title>
        <Card.Text>{item.description}</Card.Text>
      </Card.Body>
    </Card>
  );
};

export default CompanyCard;
