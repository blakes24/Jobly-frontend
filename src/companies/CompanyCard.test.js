import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router';
import CompanyCard from './CompanyCard';

test('renders without crashing', () => {
  render(
    <MemoryRouter>
      <CompanyCard />
    </MemoryRouter>
  );
});

it('matches snapshot', function() {
  const { asFragment } = render(
    <MemoryRouter>
      <CompanyCard />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
