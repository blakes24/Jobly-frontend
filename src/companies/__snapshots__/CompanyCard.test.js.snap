// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="CompanyCard card"
  >
    <a
      href="/companies/undefined"
    >
      <div
        class="card-body"
      >
        <div
          class="CompanyCard-title"
        >
          <div
            class="card-title h5"
          />
          <img
            alt="logo"
            height="25px"
          />
        </div>
        <p
          class="card-text"
        />
      </div>
    </a>
  </div>
</DocumentFragment>
`;
